main.go

```diff
@@ -70,6 +70,7 @@ func init() {
 
 func main() {
 	flag.Parse()
+	audit.CheckDeprecatedFlags()
```

Comment:
We should move this to the `New()` function in the audit manager.

This will decouple the flag deprecation from the rest of the library.

Reply:
:+1: done